dist/bin/kctf-cluster

```diff
@@ -36,6 +39,23 @@ function create_operator {
   done
 }
 
+function wait_for_nameserver {
+  nameserver="$1"
+  initial_timeout=180
+  timeout=$initial_timeout
+  sleep_time=10
+  while [[ "${timeout}" -gt 0 ]]; do
+    if nslookup -nosearch -norecurse -type=NS "${DOMAIN_NAME}." "${nameserver}" >/dev/null 2>/dev/null; then
```

Comment:
In my tests, the updates arrive faster than the negative ttl. So I think the google servers don't cache the negative response in this instance.
Also this would be a 1m delay worst case, so leaving as is for now. Please reopen if you disagree